cmd/fever/cmds/run.go

```diff
@@ -219,6 +219,10 @@ func mainfunc(cmd *cobra.Command, args []string) {
 				log.Fatal(err)
 			}
 		}
+
+		addFields := viper.GetStringMapString("add-fields")
+		forwardHandler.(*processing.ForwardHandler).AddFields(addFields)
```

Comment:
`forwardHandler.(*processing.ForwardHandler)`, is done so much in the code: why not just once `fh := forwardHandler.(*processing.ForwardHandler)`?

Reply:
Ack.